packages/client-web/src/services/storage/index.ts

```diff
@@ -8,7 +8,7 @@ export const makeStorage = <Data extends object>(
   key: string,
 ): Storage<Data> => {
   const getData: Storage<Data>["getData"] = () => {
-    const data = localStorage.getItem(makeKey(key));
+    const data = sessionStorage.getItem(makeKey(key));
```

Comment:
Change to session because don't want to persist the user session indefinitely. Now it will be cleared when they close the browser